torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -223,9 +223,6 @@ def __init__(
             # Sometimes inductor configs are unpickleable and can fail
             raise BypassAOTAutogradCache from e
 
-    def debug_lines(self) -> List[str]:
```

Comment:
Couldn't figure out why we need this since we can just do `AOTAutogradCachePickler.debug_lines(details)`. Same in `codecache.py`